plugin_tests/discourse_sso_test.py

```diff
@@ -199,11 +211,13 @@ def testDiscourseLogin(self):
             ('nonce', 'email', 'external_id', 'username', 'name',
              'require_activation'))
         self.assertEqual(parsed['nonce'][0], 'cde5d95f27062eb09c98736c3f2aecce')
-        self.assertEqual(parsed['email'][0], 'user1@email.com')
-        self.assertEqual(parsed['external_id'][0], str(self.user['_id']))
-        self.assertEqual(parsed['username'][0], 'user1login')
-        self.assertEqual(parsed['name'][0], 'First Last')
+        self.assertEqual(parsed['email'][0], 'user2@email.com')
+        self.assertEqual(parsed['external_id'][0], str(adminUser['_id']))
+        self.assertEqual(parsed['username'][0], 'user2login')
+        self.assertEqual(parsed['name'][0], 'Admin Last')
         self.assertEqual(parsed['require_activation'][0], 'false')
+        self.assertEqual(parsed['admin'][0], 'true')
+        self.assertNotIn('add_groups', parsed)
```

Comment:
Why is `add_group` not present for the admin user?

Reply:
In this test case, the admin user is a member of no groups. When that gets URL-encoded, it is logically `add_groups=`, which actually gets emitted as an empty string, due to the internal rules of query strings.

Reply:
I see, thanks!